pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -299,13 +299,15 @@ stdenv.mkDerivation ((drvAttrs config stdenv.hostPlatform.platform kernelPatches
   enableParallelBuilding = true;
 
   depsBuildBuild = [ buildPackages.stdenv.cc ];
-  nativeBuildInputs = [ perl bc nettools openssl rsync gmp libmpc mpfr ]
+  nativeBuildInputs = [ perl bc nettools openssl rsync gmp libmpc mpfr gawk ]
       ++ optional  (stdenv.hostPlatform.platform.kernelTarget == "uImage") buildPackages.ubootTools
       ++ optional  (stdenv.lib.versionAtLeast version "4.14" && stdenv.lib.versionOlder version "5.8") libelf
       # Removed util-linuxMinimal since it should not be a dependency.
       ++ optionals (stdenv.lib.versionAtLeast version "4.16") [ bison flex ]
       ++ optional  (stdenv.lib.versionAtLeast version "5.2")  cpio
       ++ optional  (stdenv.lib.versionAtLeast version "5.8")  elfutils
+      # zstd is not available in stdenv
+      ++ optional  (config.isEnabled "KERNEL_ZSTD")  zstd
```

Comment:
For some reason it complaints that it cannot find `isEnabled` here.

Reply:
Ah okay, `isEnabled` is available only within drvAttrs...

https://github.com/NixOS/nixpkgs/blob/43d8486456d558fdbba77896d301541506d2c685/pkgs/os-specific/linux/kernel/manual-config.nix#L82 

Not sure what the cleanest solution is, moving this one level up? It redefines `config` though so it might break something else.

Reply:
Personally, I'd rather just see `zstd` added to stdenv.

Reply:
Yeah. Let's just added to `nativeBuildInputs` unconditionally. The expression is already quite complex.